app/src/main/java/org/standardnotes/notes/comms/Crypt.java

```diff
@@ -223,6 +223,27 @@ public static EncryptedItem encrypt(Note note) {
         return null;
     }
 
```

Comment:
Ah, ok, just saw this. I just merged those two methods into a single encrypt EncryptableItem method.